.github/workflows/CI-pipeline.yml

```diff
@@ -269,6 +269,8 @@ jobs:
       - name: Deploy to new namespace
         working-directory: ./manifests/claudie
         run: |
+          #turn on proxy
+          sed -i 's/HTTP_PROXY_MODE=default/HTTP_PROXY_MODE=on/g' .env
```

Comment:
For e2e I would prefer to have it turned on, so that we can also catch if something goes wrong. However it will use up some of the traffic so I'm not sure.

I think we should also turn off the proxy by default and let it turn on manually by users. Afaik, once we turn the proxy on we can't turn it off again since we do not unset the env variables anywhere.

@bernardhalas  what do you think ?

Reply:
I think we need to be able to allow the user to turn off the proxy if they decide to do that.
What are the cons of turning it on by default (except of the inability to turn it off afterwards)?

Reply:
I don't see any cons, besides that we can't turn it off atm.

Possibly What I might see as a con is that maybe someone stumbles upon an edge case that we don't have in our white list for allowed domains. But that could be handled be adjusting it.